include/swift/Option/Options.td

```diff
@@ -782,6 +782,11 @@ def enable_experimental_cxx_interop :
   Flags<[NoDriverOption, FrontendOption, HelpHidden, ModuleInterfaceOption]>,
   HelpText<"Enable experimental C++ interop code generation and config directives">;
 
+def cxx_safe_interop :
+  Flag<["-"], "cxx-safe-interop">,
+  Flags<[FrontendOption, ModuleInterfaceOption, SwiftSymbolGraphExtractOption]>,
+  HelpText<"Import unannotated types and functions as unsafe.">;
```

Comment:
You won't need a special parameter when you add a feature to `Features.def`, because you'll get `--enable-experimental-feature YourFeatureName` for free.